jobs/jobscheduler.py

```diff
@@ -215,8 +231,7 @@ def initJobs(jobs, resources, otype):
             total_jobs += 1
             job = group["commands"][i]
             job["origtime"] = job["time"]
-            if "HIP_VISIBLE_DEVICES=0" in job["command"]:
-                job["gpu"] = -1
+            job["gpu"] = 1 if "-w gpu" in job["command"] else -1
```

Comment:
@iarspider , `-w gpu` is runTheMatrix option. The `command` here is `cmsDriver.py` and it will not have this option

Reply:
you can see https://cmssdt.cern.ch/SDT/cgi-bin/logreader/el8_amd64_gcc12/CMSSW_15_1_ROCM_X_2025-04-06-2300/pyRelValMatrixLogs/run/141.044406_Run3-2023_JetMET2023D_RecoPixelOnlyTripletsGPU/job.json#/ for the details of `job` object

Reply:
I see. I'll find a different solution.